### PR TITLE
feat: added rating component

### PR DESCRIPTION
**Pull request by AshutoshM29**
I have raised PR for the rating component!!

### CODE REVIEW

File: Pages/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <i class="rating fas fa-star"></i>
+            <i class="rating fas fa-star"></i>
+            <i class="rating fas fa-star"></i>
+        </div>
+        <p class="body-r">Copy the given code to make create Toast with Buttons</p>
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522star-rating%2522%253E%250A%2509%253Ci%2520class%253D%2522rating%2520fas%2520fa-star%2520starred%2522%253E%253C%252Fi%253E%250A%2509%253Ci%2520class%253D%2522rating%2520fas%2520fa-star%2520starred%2522%253E%253C%252Fi%253E%250A%2509%253Ci%2520class%253D%2522rating%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2509%253Ci%2520class%253D%2522rating%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2509%253Ci%2520class%253D%2522rating%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E"
+  style="width: 50rem; height: 35rem; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
```
Comment (vanshitamoolchandani) [inline Pages/Rating/rating.html:66]: Change- Avoid inline styling

**Review by vanshitamoolchandani — APPROVED** [review]
Looks goot to me

File: Pages/Rating/rating.html (hunk 1 above)
Comment (AshutoshM29) [reply Pages/Rating/rating.html:66]: This inline style is from carbon :(

**Review by Naman-Saxena1 — APPROVED** [review]
Looks good to me, even commit message is fine 👍 
Only think I can think of is you can add screenshots in PR description for reviewer.

**Comment by AshutoshM29** [comment]
> Looks good to me, even commit message is fine 👍 Only think I can think of is you can add screenshots in PR description for reviewer.

Thank you for the review, will keep that in mind

**Pull request merged**